Random test fail on plotting functions with windows workers

## Current Behavior
Tests are failing randomly with windows workers and then pass on a rerun.
This is very strange and leads to error output like this:

```
================================== FAILURES ===================================
____________________ test_traceplot_show[cmap_segmentdata] ____________________

args = (array([-100,  -99,  -98,  -97,  -96,  -95,  -94,  -93,  -92,  -91,  -90,
        -89,  -88,  -87,  -86,  -85,  -84,  ...,   83,   84,   85,   86,
         87,   88,   89,   90,   91,   92,   93,   94,   95,   96,   97,
         98,   99]))
kwargs = {'cmap_segmentdata': {}}
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x000002D4ACECF408>

    @pytest.mark.parametrize(
        'kwargs',
        [
            pytest.param({}, id='no_kwargs'),
            pytest.param({'cval': np.arange(-100, 100)}, id='cval_array'),
            pytest.param(
                {'cval': np.arange(-100, 100), 'cmap_norm': 'twoslope'},
                id='cmap_norm_twoslope',
            ),
            pytest.param(
                {'cval': np.arange(0, 200), 'cmap_norm': 'nonorm'},
                id='cmap_norm_nonorm',
            ),
            pytest.param(
                {'cval': np.arange(0, 200)},
                id='cmap_norm_nonorm_implicit',
            ),
            pytest.param(
                {'cval': np.arange(-100, 100), 'cmap_norm': 'normalize'},
                id='cmap_norm_normalize',
            ),
            pytest.param(
                {'cval': np.arange(0, 200), 'cmap_norm': 'linear'},
                id='cmap_norm_linear',
            ),
            pytest.param(
                {'cval': np.arange(0, 200), 'cmap_norm': matplotlib.colors.NoNorm()},
                id='cmap_norm_class',
            ),
            pytest.param(
                {'cmap': matplotlib.colors.LinearSegmentedColormap(name='test', segmentdata={})},
                id='cmap_class',
            ),
            pytest.param({'cmap_segmentdata': {}}, id='cmap_segmentdata'),
            pytest.param({'padding': 0.1}, id='padding'),
            pytest.param({'cval': np.arange(0, 200), 'show_cbar': True}, id='show_cbar_true'),
            pytest.param({'cval': np.arange(0, 200), 'show_cbar': False}, id='show_cbar_false'),
        ],
    )
    def test_traceplot_show(args, kwargs, monkeypatch):
        mock = Mock()
        monkeypatch.setattr(plt, 'show', mock)
>       traceplot(*args, **kwargs)

args       = (array([-100,  -99,  -98,  -97,  -96,  -95,  -94,  -93,  -92,  -91,  -90,
        -89,  -88,  -87,  -86,  -85,  -84,  -83,  -82,  -81,  -80,  -79,
        -78,  -77,  -76,  -75,  -74,  -73,  -72,  -71,  -70,  -69,  -68,
        -67,  -66,  -65,  -64,  -63,  -62,  -61,  -60,  -59,  -58,  -57,
        -56,  -55,  -54,  -53,  -52,  -51,  -50,  -49,  -48,  -47,  -46,
        -45,  -44,  -43,  -42,  -41,  -40,  -39,  -38,  -37,  -36,  -35,
        -34,  -33,  -32,  -31,  -30,  -29,  -28,  -27,  -26,  -25,  -24,
        -23,  -22,  -21,  -20,  -19,  -18,  -17,  -16,  -15,  -14,  -13,
        -12,  -11,  -10,   -9,   -8,   -7,   -6,   -5,   -4,   -3,   -2,
         -1,    0,    1,    2,    3,    4,    5,    6,    7,    8,    9,
         10,   11,   12,   13,   14,   15,   16,   17,   18,   19,   20,
         21,   22,   23,   24,   25,   26,   27,   28,   29,   30,   31,
         32,   33,   34,   35,   36,   37,   38,   39,   40,   41,   42,
         43,   44,   45,   46,   47,   48,   49,   50,   51,   52,   53,
         54,   55,   56,   57,   58,   59,   60,   61,   62,   63,   64,
         65,   66,   67,   68,   69,   70,   71,   72,   73,   74,   75,
         76,   77,   78,   79,   80,   81,   82,   83,   84,   85,   86,
         87,   88,   89,   90,   91,   92,   93,   94,   95,   96,   97,
         98,   99]),
 array([-100,  -99,  -98,  -97,  -96,  -95,  -94,  -93,  -92,  -91,  -90,
        -89,  -88,  -87,  -86,  -85,  -84,  -83,  -82,  -81,  -80,  -79,
        -78,  -77,  -76,  -75,  -74,  -73,  -72,  -71,  -70,  -69,  -68,
        -67,  -66,  -65,  -64,  -63,  -62,  -61,  -60,  -59,  -58,  -57,
        -56,  -55,  -54,  -53,  -52,  -51,  -50,  -49,  -48,  -47,  -46,
        -45,  -44,  -43,  -42,  -41,  -40,  -39,  -38,  -37,  -36,  -35,
        -34,  -33,  -32,  -31,  -30,  -29,  -28,  -27,  -26,  -25,  -24,
        -23,  -22,  -21,  -20,  -19,  -18,  -17,  -16,  -15,  -14,  -13,
        -12,  -11,  -10,   -9,   -8,   -7,   -6,   -5,   -4,   -3,   -2,
         -1,    0,    1,    2,    3,    4,    5,    6,    7,    8,    9,
         10,   11,   12,   13,   14,   15,   16,   17,   18,   19,   20,
         21,   22,   23,   24,   25,   26,   27,   28,   29,   30,   31,
         32,   33,   34,   35,   36,   37,   38,   39,   40,   41,   42,
         43,   44,   45,   46,   47,   48,   49,   50,   51,   52,   53,
         54,   55,   56,   57,   58,   59,   60,   61,   62,   63,   64,
         65,   66,   67,   68,   69,   70,   71,   72,   73,   74,   75,
         76,   77,   78,   79,   80,   81,   82,   83,   84,   85,   86,
         87,   88,   89,   90,   91,   92,   93,   94,   95,   96,   97,
         98,   99]))
kwargs     = {'cmap_segmentdata': {}}
mock       = <Mock id='3112215334984'>
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x000002D4ACECF408>

tests\plotting\traceplot_test.py:82: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
src\pymovements\plotting\traceplot.py:137: in traceplot
    fig = plt.figure(figsize=figsize)
        cbar_label = None
        cmap       = None
        cmap_norm  = None
        cmap_segmentdata = {}
        cval       = None
        figsize    = (15, 5)
        n          = 200
        pad_factor = 0.05
        padding    = None
        savepath   = None
        show       = True
        show_cbar  = False
        title      = None
        x          = array([-100,  -99,  -98,  -97,  -96,  -95,  -94,  -93,  -92,  -91,  -90,
        -89,  -88,  -87,  -86,  -85,  -84,  -83,  -82,  -81,  -80,  -79,
        -78,  -77,  -76,  -75,  -74,  -73,  -72,  -71,  -70,  -69,  -68,
        -67,  -66,  -65,  -64,  -63,  -62,  -61,  -60,  -59,  -58,  -57,
        -56,  -55,  -54,  -53,  -52,  -51,  -50,  -49,  -48,  -47,  -46,
        -45,  -44,  -43,  -42,  -41,  -40,  -39,  -38,  -37,  -36,  -35,
        -34,  -33,  -32,  -31,  -30,  -29,  -28,  -27,  -26,  -25,  -24,
        -23,  -22,  -21,  -20,  -19,  -18,  -17,  -16,  -15,  -14,  -13,
        -12,  -11,  -10,   -9,   -8,   -7,   -6,   -5,   -4,   -3,   -2,
         -1,    0,    1,    2,    3,    4,    5,    6,    7,    8,    9,
         10,   11,   12,   13,   14,   15,   16,   17,   18,   19,   20,
         21,   22,   23,   24,   25,   26,   27,   28,   29,   30,   31,
         32,   33,   34,   35,   36,   37,   38,   39,   40,   41,   42,
         43,   44,   45,   46,   47,   48,   49,   50,   51,   52,   53,
         54,   55,   56,   57,   58,   59,   60,   61,   62,   63,   64,
         65,   66,   67,   68,   69,   70,   71,   72,   73,   74,   75,
         76,   77,   78,   79,   80,   81,   82,   83,   84,   85,   86,
         87,   88,   89,   90,   91,   92,   93,   94,   95,   96,   97,
         98,   99])
        y          = array([-100,  -99,  -98,  -97,  -96,  -95,  -94,  -93,  -92,  -91,  -90,
        -89,  -88,  -87,  -86,  -85,  -84,  -83,  -82,  -81,  -80,  -79,
        -78,  -77,  -76,  -75,  -74,  -73,  -72,  -71,  -70,  -69,  -68,
        -67,  -66,  -65,  -64,  -63,  -62,  -61,  -60,  -59,  -58,  -57,
        -56,  -55,  -54,  -53,  -52,  -51,  -50,  -49,  -48,  -47,  -46,
        -45,  -44,  -43,  -42,  -41,  -40,  -39,  -38,  -37,  -36,  -35,
        -34,  -33,  -32,  -31,  -30,  -29,  -28,  -27,  -26,  -25,  -24,
        -23,  -22,  -21,  -20,  -19,  -18,  -17,  -16,  -15,  -14,  -13,
        -12,  -11,  -10,   -9,   -8,   -7,   -6,   -5,   -4,   -3,   -2,
         -1,    0,    1,    2,    3,    4,    5,    6,    7,    8,    9,
         10,   11,   12,   13,   14,   15,   16,   17,   18,   19,   20,
         21,   22,   23,   24,   25,   26,   27,   28,   29,   30,   31,
         32,   33,   34,   35,   36,   37,   38,   39,   40,   41,   42,
         43,   44,   45,   46,   47,   48,   49,   50,   51,   52,   53,
         54,   55,   56,   57,   58,   59,   60,   61,   62,   63,   64,
         65,   66,   67,   68,   69,   70,   71,   72,   73,   74,   75,
         76,   77,   78,   79,   80,   81,   82,   83,   84,   85,   86,
         87,   88,   89,   90,   91,   92,   93,   94,   95,   96,   97,
         98,   99])
.tox\py37\lib\site-packages\matplotlib\pyplot.py:809: in figure
    FigureClass=FigureClass, **kwargs)
        FigureClass = <class 'matplotlib.figure.Figure'>
        allnums    = []
        clear      = False
        dpi        = None
        edgecolor  = None
        facecolor  = None
        fig_label  = ''
        figsize    = (15, 5)
        frameon    = True
        kwargs     = {}
        manager    = None
        max_open_warning = 20
        next_num   = 1
        num        = 1
.tox\py37\lib\site-packages\matplotlib\pyplot.py:325: in new_figure_manager
    return _get_backend_mod().new_figure_manager(*args, **kwargs)
        args       = (1,)
        kwargs     = {'FigureClass': <class 'matplotlib.figure.Figure'>,
 'dpi': None,
 'edgecolor': None,
 'facecolor': None,
 'figsize': (15, 5),
 'frameon': True}
.tox\py37\lib\site-packages\matplotlib\backend_bases.py:3494: in new_figure_manager
    return cls.new_figure_manager_given_figure(num, fig)
        Figure     = <class 'matplotlib.figure.Figure'>
        args       = ()
        cls        = <class 'matplotlib.backends.backend_tkagg._BackendTkAgg'>
        fig        = <Figure size 1500x500 with 0 Axes>
        fig_cls    = <class 'matplotlib.figure.Figure'>
        kwargs     = {'dpi': None,
 'edgecolor': None,
 'facecolor': None,
 'figsize': (15, 5),
 'frameon': True}
        num        = 1
.tox\py37\lib\site-packages\matplotlib\backends\_backend_tk.py:949: in new_figure_manager_given_figure
    window = tk.Tk(className="matplotlib")
        cls        = <class 'matplotlib.backends.backend_tkagg._BackendTkAgg'>
        figure     = <Figure size 1500x500 with 0 Axes>
        num        = 1
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tkinter.Tk object .>, screenName = None, baseName = 'pytest.EXE'
className = 'matplotlib', useTk = 1, sync = 0, use = None

    def __init__(self, screenName=None, baseName=None, className='Tk',
                 useTk=1, sync=0, use=None):
        """Return a new Toplevel widget on screen SCREENNAME. A new Tcl interpreter will
        be created. BASENAME will be used for the identification of the profile file (see
        readprofile).
        It is constructed from sys.argv[0] without extensions if None is given. CLASSNAME
        is the name of the widget class."""
        self.master = None
        self.children = {}
        self._tkloaded = 0
        # to avoid recursions in the getattr code in case of failure, we
        # ensure that self.tk is always _something_.
        self.tk = None
        if baseName is None:
            import os
            baseName = os.path.basename(sys.argv[0])
            baseName, ext = os.path.splitext(baseName)
            if ext not in ('.py', '.pyc'):
                baseName = baseName + ext
        interactive = 0
>       self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
E       _tkinter.TclError: Can't find a usable init.tcl in the following directories: 
E           {C:\hostedtoolcache\windows\Python\3.7.9\x64\tcl\tcl8.6}
E       
E       C:/hostedtoolcache/windows/Python/3.7.9/x64/tcl/tcl8.6/init.tcl: couldn't read file "C:/hostedtoolcache/windows/Python/3.7.9/x64/tcl/tcl8.6/init.tcl": No error
E       couldn't read file "C:/hostedtoolcache/windows/Python/3.7.9/x64/tcl/tcl8.6/init.tcl": No error
E           while executing
E       "source C:/hostedtoolcache/windows/Python/3.7.9/x64/tcl/tcl8.6/init.tcl"
E           ("uplevel" body line 1)
E           invoked from within
E       "uplevel #0 [list source $tclfile]"
E       
E       
E       This probably means that Tcl wasn't installed properly.

baseName   = 'pytest.EXE'
className  = 'matplotlib'
ext        = '.EXE'
interactive = 0
os         = <module 'os' from 'C:\\hostedtoolcache\\windows\\Python\\3.7.9\\x64\\lib\\os.py'>
screenName = None
self       = <tkinter.Tk object .>
sync       = 0
use        = None
useTk      = 1

C:\hostedtoolcache\windows\Python\3.7.9\x64\lib\tkinter\__init__.py:2023: TclError
```

## Expected Behavior
Tests should run deterministically and shouldn't "just work" after a rerun.

### Steps to Reproduce
The bug is very hard to reproduce as it occurs randomly.

Again on main with `traceplot`

```
================================== FAILURES ===================================
_____________________________ test_traceplot_save _____________________________

args = (array([-100,  -99,  -98,  -97,  -96,  -95,  -94,  -93,  -92,  -91,  -90,
        -89,  -88,  -87,  -86,  -85,  -84,  ...,   83,   84,   85,   86,
         87,   88,   89,   90,   91,   92,   93,   94,   95,   96,   97,
         98,   99]))
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x0000018BCF3AA620>
tmp_path = WindowsPath('C:/Users/runneradmin/AppData/Local/Temp/pytest-of-unknown/pytest-0/test_traceplot_save0')

    def test_traceplot_save(args, monkeypatch, tmp_path):
        mock = Mock()
        monkeypatch.setattr(figure.Figure, 'savefig', mock)
>       traceplot(*args, show=False, savepath=str(tmp_path / 'test.svg'))

args       = (array([-100,  -99,  -98,  -97,  -96,  -95,  -94,  -93,  -92,  -91,  -90,
        -89,  -88,  -87,  -86,  -85,  -84,  -83,  -82,  -81,  -80,  -79,
        -78,  -77,  -76,  -75,  -74,  -73,  -72,  -71,  -70,  -69,  -68,
        -67,  -66,  -65,  -64,  -63,  -62,  -61,  -60,  -59,  -58,  -57,
        -56,  -55,  -54,  -53,  -52,  -51,  -50,  -49,  -48,  -47,  -46,
        -45,  -44,  -43,  -42,  -41,  -40,  -39,  -38,  -37,  -36,  -35,
        -34,  -33,  -32,  -31,  -30,  -29,  -28,  -27,  -26,  -25,  -24,
        -23,  -22,  -21,  -20,  -19,  -18,  -17,  -16,  -15,  -14,  -13,
        -12,  -11,  -10,   -9,   -8,   -7,   -6,   -5,   -4,   -3,   -2,
         -1,    0,    1,    2,    3,    4,    5,    6,    7,    8,    9,
         10,   11,   12,   13,   14,   15,   16,   17,   18,   19,   20,
         21,   22,   23,   24,   25,   26,   27,   28,   29,   30,   31,
         32,   33,   34,   35,   36,   37,   38,   39,   40,   41,   42,
         43,   44,   45,   46,   47,   48,   49,   50,   51,   52,   53,
         54,   55,   56,   57,   58,   59,   60,   61,   62,   63,   64,
         65,   66,   67,   68,   69,   70,   71,   72,   73,   74,   75,
         76,   77,   78,   79,   80,   81,   82,   83,   84,   85,   86,
         87,   88,   89,   90,   91,   92,   93,   94,   95,   96,   97,
         98,   99]),
 array([-100,  -99,  -98,  -97,  -96,  -95,  -94,  -93,  -92,  -91,  -90,
        -89,  -88,  -87,  -86,  -85,  -84,  -83,  -82,  -81,  -80,  -79,
        -78,  -77,  -76,  -75,  -74,  -73,  -72,  -71,  -70,  -69,  -68,
        -67,  -66,  -65,  -64,  -63,  -62,  -61,  -60,  -59,  -58,  -57,
        -56,  -55,  -54,  -53,  -52,  -51,  -50,  -49,  -48,  -47,  -46,
        -45,  -44,  -43,  -42,  -41,  -40,  -39,  -38,  -37,  -36,  -35,
        -34,  -33,  -32,  -31,  -30,  -29,  -28,  -27,  -26,  -25,  -24,
        -23,  -22,  -21,  -20,  -19,  -18,  -17,  -16,  -15,  -14,  -13,
        -12,  -11,  -10,   -9,   -8,   -7,   -6,   -5,   -4,   -3,   -2,
         -1,    0,    1,    2,    3,    4,    5,    6,    7,    8,    9,
         10,   11,   12,   13,   14,   15,   16,   17,   18,   19,   20,
         21,   22,   23,   24,   25,   26,   27,   28,   29,   30,   31,
         32,   33,   34,   35,   36,   37,   38,   39,   40,   41,   42,
         43,   44,   45,   46,   47,   48,   49,   50,   51,   52,   53,
         54,   55,   56,   57,   58,   59,   60,   61,   62,   63,   64,
         65,   66,   67,   68,   69,   70,   71,   72,   73,   74,   75,
         76,   77,   78,   79,   80,   81,   82,   83,   84,   85,   86,
         87,   88,   89,   90,   91,   92,   93,   94,   95,   96,   97,
         98,   99]))
mock       = <Mock id='1699988806176'>
monkeypatch = <_pytest.monkeypatch.MonkeyPatch object at 0x0000018BCF3AA620>
tmp_path   = WindowsPath('C:/Users/runneradmin/AppData/Local/Temp/pytest-of-unknown/pytest-0/test_traceplot_save0')

tests\plotting\traceplot_test.py:98: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
src\pymovements\plotting\traceplot.py:137: in traceplot
    fig = plt.figure(figsize=figsize)
        cbar_label = None
        cmap       = None
        cmap_norm  = None
        cmap_segmentdata = None
        cval       = None
        figsize    = (15, 5)
        n          = 200
        pad_factor = 0.05
        padding    = None
        savepath   = 'C:\\Users\\runneradmin\\AppData\\Local\\Temp\\pytest-of-unknown\\pytest-0\\test_traceplot_save0\\test.svg'
        show       = False
        show_cbar  = False
        title      = None
        x          = array([-100,  -99,  -98,  -97,  -96,  -95,  -94,  -93,  -92,  -91,  -90,
        -89,  -88,  -87,  -86,  -85,  -84,  -83,  -82,  -81,  -80,  -79,
        -78,  -77,  -76,  -75,  -74,  -73,  -72,  -71,  -70,  -69,  -68,
        -67,  -66,  -65,  -64,  -63,  -62,  -61,  -60,  -59,  -58,  -57,
        -56,  -55,  -54,  -53,  -52,  -51,  -50,  -49,  -48,  -47,  -46,
        -45,  -44,  -43,  -42,  -41,  -40,  -39,  -38,  -37,  -36,  -35,
        -34,  -33,  -32,  -31,  -30,  -29,  -28,  -27,  -26,  -25,  -24,
        -23,  -22,  -21,  -20,  -19,  -18,  -17,  -16,  -15,  -14,  -13,
        -12,  -11,  -10,   -9,   -8,   -7,   -6,   -5,   -4,   -3,   -2,
         -1,    0,    1,    2,    3,    4,    5,    6,    7,    8,    9,
         10,   11,   12,   13,   14,   15,   16,   17,   18,   19,   20,
         21,   22,   23,   24,   25,   26,   27,   28,   29,   30,   31,
         32,   33,   34,   35,   36,   37,   38,   39,   40,   41,   42,
         43,   44,   45,   46,   47,   48,   49,   50,   51,   52,   53,
         54,   55,   56,   57,   58,   59,   60,   61,   62,   63,   64,
         65,   66,   67,   68,   69,   70,   71,   72,   73,   74,   75,
         76,   77,   78,   79,   80,   81,   82,   83,   84,   85,   86,
         87,   88,   89,   90,   91,   92,   93,   94,   95,   96,   97,
         98,   99])
        y          = array([-100,  -99,  -98,  -97,  -96,  -95,  -94,  -93,  -92,  -91,  -90,
        -89,  -88,  -87,  -86,  -85,  -84,  -83,  -82,  -81,  -80,  -79,
        -78,  -77,  -76,  -75,  -74,  -73,  -72,  -71,  -70,  -69,  -68,
        -67,  -66,  -65,  -64,  -63,  -62,  -61,  -60,  -59,  -58,  -57,
        -56,  -55,  -54,  -53,  -52,  -51,  -50,  -49,  -48,  -47,  -46,
        -45,  -44,  -43,  -42,  -41,  -40,  -39,  -38,  -37,  -36,  -35,
        -34,  -33,  -32,  -31,  -30,  -29,  -28,  -27,  -26,  -25,  -24,
        -23,  -22,  -21,  -20,  -19,  -18,  -17,  -16,  -15,  -14,  -13,
        -12,  -11,  -10,   -9,   -8,   -7,   -6,   -5,   -4,   -3,   -2,
         -1,    0,    1,    2,    3,    4,    5,    6,    7,    8,    9,
         10,   11,   12,   13,   14,   15,   16,   17,   18,   19,   20,
         21,   22,   23,   24,   25,   26,   27,   28,   29,   30,   31,
         32,   33,   34,   35,   36,   37,   38,   39,   40,   41,   42,
         43,   44,   45,   46,   47,   48,   49,   50,   51,   52,   53,
         54,   55,   56,   57,   58,   59,   60,   61,   62,   63,   64,
         65,   66,   67,   68,   69,   70,   71,   72,   73,   74,   75,
         76,   77,   78,   79,   80,   81,   82,   83,   84,   85,   86,
         87,   88,   89,   90,   91,   92,   93,   94,   95,   96,   97,
         98,   99])
.tox\py310\lib\site-packages\matplotlib\_api\deprecation.py:454: in wrapper
    return func(*args, **kwargs)
        args       = ()
        func       = <function figure at 0x0000018BBBD637F0>
        kwargs     = {'figsize': (15, 5)}
        name       = 'facecolor'
        name_idx   = 3
        since      = '3.6'
.tox\py310\lib\site-packages\matplotlib\pyplot.py:840: in figure
    manager = new_figure_manager(
        FigureClass = <class 'matplotlib.figure.Figure'>
        allnums    = []
        clear      = False
        dpi        = None
        edgecolor  = None
        facecolor  = None
        fig_label  = ''
        figsize    = (15, 5)
        frameon    = True
        kwargs     = {}
        manager    = None
        max_open_warning = 20
        next_num   = 1
        num        = 1
.tox\py310\lib\site-packages\matplotlib\pyplot.py:384: in new_figure_manager
    return _get_backend_mod().new_figure_manager(*args, **kwargs)
        args       = (1,)
        kwargs     = {'FigureClass': <class 'matplotlib.figure.Figure'>,
 'dpi': None,
 'edgecolor': None,
 'facecolor': None,
 'figsize': (15, 5),
 'frameon': True}
.tox\py310\lib\site-packages\matplotlib\backend_bases.py:3574: in new_figure_manager
    return cls.new_figure_manager_given_figure(num, fig)
        Figure     = <class 'matplotlib.figure.Figure'>
        args       = ()
        cls        = <class 'matplotlib.backends.backend_tkagg._BackendTkAgg'>
        fig        = <Figure size 1500x500 with 0 Axes>
        fig_cls    = <class 'matplotlib.figure.Figure'>
        kwargs     = {'dpi': None,
 'edgecolor': None,
 'facecolor': None,
 'figsize': (15, 5),
 'frameon': True}
        num        = 1
.tox\py310\lib\site-packages\matplotlib\backend_bases.py:3579: in new_figure_manager_given_figure
    return cls.FigureCanvas.new_manager(figure, num)
        cls        = <class 'matplotlib.backends.backend_tkagg._BackendTkAgg'>
        figure     = <Figure size 1500x500 with 0 Axes>
        num        = 1
.tox\py310\lib\site-packages\matplotlib\backend_bases.py:1742: in new_manager
    return cls.manager_class.create_with_canvas(cls, figure, num)
        cls        = <class 'matplotlib.backends.backend_tkagg.FigureCanvasTkAgg'>
        figure     = <Figure size 1500x500 with 0 Axes>
        num        = 1
.tox\py310\lib\site-packages\matplotlib\backends\_backend_tk.py:483: in create_with_canvas
    window = tk.Tk(className="matplotlib")
        canvas_class = <class 'matplotlib.backends.backend_tkagg.FigureCanvasTkAgg'>
        cls        = <class 'matplotlib.backends._backend_tk.FigureManagerTk'>
        figure     = <Figure size 1500x500 with 0 Axes>
        num        = 1
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <tkinter.Tk object .>, screenName = None, baseName = 'pytest.EXE'
className = 'matplotlib', useTk = True, sync = False, use = None

    def __init__(self, screenName=None, baseName=None, className='Tk',
                 useTk=True, sync=False, use=None):
        """Return a new top level widget on screen SCREENNAME. A new Tcl interpreter will
        be created. BASENAME will be used for the identification of the profile file (see
        readprofile).
        It is constructed from sys.argv[0] without extensions if None is given. CLASSNAME
        is the name of the widget class."""
        self.master = None
        self.children = {}
        self._tkloaded = False
        # to avoid recursions in the getattr code in case of failure, we
        # ensure that self.tk is always _something_.
        self.tk = None
        if baseName is None:
            import os
            baseName = os.path.basename(sys.argv[0])
            baseName, ext = os.path.splitext(baseName)
            if ext not in ('.py', '.pyc'):
                baseName = baseName + ext
        interactive = False
>       self.tk = _tkinter.create(screenName, baseName, className, interactive, wantobjects, useTk, sync, use)
E       _tkinter.TclError: Can't find a usable tk.tcl in the following directories: 
E           C:/hostedtoolcache/windows/Python/3.10.11/x64/tcl/tcl8.6/tk8.6 C:/hostedtoolcache/windows/Python/3.10.11/x64/tcl/tk8.6 C:/hostedtoolcache/windows/Python/3.10.11/lib/tk8.6 C:/hostedtoolcache/windows/Python/3.10.11/lib/tk8.6 C:/hostedtoolcache/windows/Python/lib/tk8.6 C:/hostedtoolcache/windows/Python/3.10.11/library
E       
E       C:/hostedtoolcache/windows/Python/3.10.11/x64/tcl/tk8.6/tk.tcl: couldn't read file "C:/hostedtoolcache/windows/Python/3.10.11/x64/tcl/tk8.6/ttk/combobox.tcl": no such file or directory
E       couldn't read file "C:/hostedtoolcache/windows/Python/3.10.11/x64/tcl/tk8.6/ttk/combobox.tcl": no such file or directory
E           while executing
E       "source -encoding utf-8 [file join $::ttk::library combobox.tcl]	"
E           (file "C:/hostedtoolcache/windows/Python/3.10.11/x64/tcl/tk8.6/ttk/ttk.tcl" line 108)
E           invoked from within
E       "source -encoding utf-8 C:/hostedtoolcache/windows/Python/3.10.11/x64/tcl/tk8.6/ttk/ttk.tcl"
E           ("uplevel" body line 1)
E           invoked from within
E       "uplevel \#0 [list source -encoding utf-8 $::ttk::library/ttk.tcl]"
E           (file "C:/hostedtoolcache/windows/Python/3.10.11/x64/tcl/tk8.6/tk.tcl" line 706)
E           invoked from within
E       "source C:/hostedtoolcache/windows/Python/3.10.11/x64/tcl/tk8.6/tk.tcl"
E           ("uplevel" body line 1)
E           invoked from within
E       "uplevel #0 [list source $file]"
E       
E       
E       This probably means that tk wasn't installed properly.

baseName   = 'pytest.EXE'
className  = 'matplotlib'
ext        = '.EXE'
interactive = False
os         = <module 'os' from 'C:\\hostedtoolcache\\windows\\Python\\3.10.11\\x64\\lib\\os.py'>
screenName = None
self       = <tkinter.Tk object .>
sync       = False
use        = None
useTk      = True

C:\hostedtoolcache\windows\Python\3.10.11\x64\lib\tkinter\__init__.py:2299: TclError
```

--- a/src/pymovements/plotting/traceplot.py
+++ b/src/pymovements/plotting/traceplot.py
@@ -31,6 +31,8 @@ from matplotlib.collections import LineCollection
 from pymovements.gaze.gaze_dataframe import GazeDataFrame
 
 
+matplotlib.use('Agg')
+
 DEFAULT_SEGMENTDATA = {
     'red': [
         [0.0, 0.0, 0.0],

